faker/providers/python/__init__.py

```diff
@@ -69,10 +70,13 @@ def pyfloat(self, left_digits=None, right_digits=None, positive=False,
         if positive and min_value is not None and min_value <= 0:
             raise ValueError(
                 'Cannot combine positive=True with negative or zero min_value')
+        if left_digits is not None and max_value is not None and math.ceil(math.log10(max_value)) > left_digits:
```

Comment:
I just realised this doesn't allow `max_value=0` (log will raise an exception) - and in that case, or actually in any case, this should be checking the larger absolute value of min or max, not just max_value.

I'll fix that (and a test, since it's clearly not covered) later.

Reply:
Fixed